Solutions/NetClean ProActive/Data Connectors/Connector_NetClean.json

```diff
@@ -0,0 +1,115 @@
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+        
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description": ">**NOTE:** The data connector relies on Azure Logic Apps to recive and push data to Log Analytics This might result in additional data ingestion costs.\n It's possible to test this without Logic Apps or NetClean Proactive see option 2",
+            "instructions": [
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Workspace ID"
+                    },
+                    "type": "CopyableLabel"
+                },
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "PrimaryKey"
+                        ],
+                        "label": "Primary Key"
+                    },
+                    "type": "CopyableLabel"
+                },
+                { 
+                    "title": " Option 1: deploy Logic app", 
+                    "description": "1.Search for and deploy the latest version of the NetClean Logic app \n 2.Modify it in accordance with the instructions found on my.netclean.com \n Copy the HTTP POST URL \n 3. Add it to the webhook configuration in your NetClean ProActive Webconsole. "
+                },
+                { 
+                    "title": " Option 2", 
+                    "description": "Ingest data using a api function. please use the script found on\n https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell \n replace the CustomerId and SharedKey values. \n replace the data in $json variable to the sample data. \n set the LogType varible to **Netclean_Incidents_CL** \n run the script"
+                }
+
+            ]
+        }
+    ],
+    "metadata": {
+        "id": "d6a16a5e-19c0-4599-bbbc-04d6c38b00d4",
+        "version": "1.0.0",
+        "kind": "Solution",
```

Comment:
Hi @NCsteven Please set the kind to "dataConnector" here in accordance with the template - [Connector Template](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json), thanks!